sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.36" />
```

Comment:
I believe eventually the version will be upped to be this. So this will most likely not be necessary.

For future reference, package reference versions are specified in the eng/Packages.Data.props file.

Reply:
Got it, thanks will update when the package has been officially released in nuget

Reply:
Updated this value globally here